common/religion/religions/shathgral_group.txt

```diff
@@ -289,7 +322,14 @@
 		ghuun_worship = {
 			#Main Group
 			color = rgb { 60 10 10 }
+			religious_head = d_ghuun_worship
 			icon = custom_faith_1
+
+			holy_site = ahnqiraj
+			holy_site = zulnazman
+			holy_site = ulduar
+			holy_site = storm_isle
+			holy_site = lulkail
```

Comment:
I suggest replacing 3 of them with 3 seals of g'huun

Reply:
Changed holy sites in 65ebb00e59743b59df66b529d72661a0d6133aa1.